Tests/OTelTests/Logging/OTelBatchLogProcessorTests.swift

```diff
@@ -0,0 +1,168 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2024 Moritz Lang and the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@testable import Logging
+@_spi(Logging) import OTel
+import OTelTesting
+import XCTest
+
+final class OTelBatchLogProcessorTests: XCTestCase {
```

Comment:
```suggestion
final class OTelBatchLogRecordProcessorTests: XCTestCase {
```

The file should be renamed too.